querybook/server/datasources/comment.py

```diff
@@ -67,21 +67,22 @@ def add_thread_comment(parent_comment_id: int, text):
 
 
 @register(
-    "/comment/<int:comment_id>/",
+    "/comment/<int:comment_id>/text/",
     methods=["PUT"],
 )
-def edit_comment(comment_id: int, **fields):
+def edit_comment_text(comment_id: int, **fields):
     assert_can_edit_and_delete(comment_id=comment_id)
     return logic.edit_comment(comment_id=comment_id, **fields)
 
 
 @register(
-    "/comment/<int:comment_id>/",
-    methods=["DELETE"],
+    "/comment/<int:comment_id>/archive/",
+    methods=["PUT"],
 )
-def remove_comment(comment_id: int):
+def archive_comment(comment_id: int, **fields):
     assert_can_edit_and_delete(comment_id=comment_id)
-    return logic.remove_comment(comment_id=comment_id)
+    logic.edit_comment(comment_id=comment_id, **fields)
+    return
```

Comment:
the comment update & soft delete in CommentResource are using the previous `edit_comment` endpoint, these two are not used.

Reply:
I am a little bit confused, why can't we use the same methods before, but change remove comment to soft delete?